kani-driver/src/harness_runner.rs

```diff
@@ -173,26 +172,22 @@ impl KaniSession {
                     "Complete - {succeeding} successfully verified harnesses, {failing} failures, {total} total."
                 );
             } else {
-                match (self.args.harnesses.as_slice(), &self.args.function) {
-                    ([], None) =>
+                match self.args.harnesses.as_slice() {
+                    [] =>
```

Comment:
Nit: I wonder if the code would be easier to read if we just do an `if ... then ... else` on the length of `self.args.harnesses`.

Reply:
I don't think it makes much difference